prisma/schema.prisma

```diff
@@ -129,3 +129,9 @@ model Message {
   @@index([createdAt])
   @@index([updatedAt])
 }
+
+model WebHookIntegrationSecret {
+  id                        Int         @default(autoincrement()) @id
+  createdAt                 DateTime    @default(now())
+  secret                    String      @db.VarChar(255)
+}
```

Comment:
I think, it might serve more purpose in the future to have the table be more generic, e.g `IntegrationKey` (as-in not specific to webhooks)

Could look something like

```prisma
model IntegrationKey {
  id                   Int         @default(autoincrement()) @id
  createdAt.           DateTime    @default(now())
  type                 String      @db.VarChar(255) @unique
  value                String      @db.VarChar(255)
} 
```

And we store it as 

```json
{
  "type": "webhook-secret",
  "value": "abc"
}
```

That would let us save other keys in the database (I can definitely imagine the smooch app ID being manually configured by the user and stored in the database for example)

Thoughts @steveyken  ?

Reply:
Sounds like sensible idea. Lots of frameworks and apps now have ways to store secrets. Seems to make sense to have a generic setup we could re-use.

Overall, great work!

Reply:
@patrixr so for any given type there can only be a single secret associated with it right? Is that why you specified `@unique`?

Reply:
yes